src/components/CopyToClipboard/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './CopyToClipboard';
-export {CopyToClipboardStatus} from './types';
+export {type CopyToClipboardStatus} from './types';
```

Comment:
Lets rewrite this export in this way:
```typescript
export type {CopyToClipboardStatus} from './types'
```
This syntax [have released in version 4.5](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names). This export will crushed code that uses typescript below version